docs/secure-connections/sauce-connect/setup-configuration/yaml-config.md

```diff
@@ -22,59 +75,42 @@ Another way to launch a tunnel is to fill out our YAML config file template and
 
 We recommend using a YAML config file in production environments.
 * **Facilitates tracking tunnel configuration changes** because they're all included in a single YAML file.
-* **Facilitates management of tunnel-domains and direct-domains options**, which can get very long.
+* **Facilitates management of potentially long CLI options such as tunnel-domains, direct-domains, etc...**
```

Comment:
@waggledans  Need to remove "etc...", so that it says:

**Facilitates management of potentially long CLI options such as `tunnel-domains` and `direct-domains`.